demo_processes/new_customer/src/main/resources/scripts/save_data.groovy

```diff
@@ -0,0 +1 @@
+// save data to Hydra
```

Comment:
For what?

Reply:
I was told to make it with a view to the saving all that stuff to Hydra.
I made this script so that the next does not edit `.bpmn` file.